### PR TITLE
[Localization] Added Korean Translation for the dialogues of the Cilan, Chili, and Cress

### PR DESCRIPTION
**Pull request by soy0ka**
## What are the changes?

Added Korean Translation for the dialogues of the Cilan, Chili, and Cress who are the Striaton Gym Leaders in Unova Region

## Why am I doing these changes?

Missing Korean Locales for those three trainers

### Before PR Changes
- Cilan, Chili, and Cress’s dialogues were not translated into Korean.
- Original dialogues were only available in English.

### After PR Changes
- Korean translations have been added for the dialogues of Cilan, Chili, and Cress, the Striaton Gym Leaders in the Unova Region.
- Both English and Korean versions of the dialogues are now available.

### Summary of Changes
- Added Korean translations for Cilan, Chili, and Cress’s dialogues.
- Updated the documentation to include both English and Korean versions of the dialogues.

## Checklist
- [X] There is no overlap with another PR?
- [X] The PR is self-contained and cannot be split into smaller PRs?
- [X] Have I provided a clear explanation of the changes?
- [X] Have I tested the changes (manually)?
    - [X] Are all unit tests still passing? (`npm run test`)
- [X] Are the changes visual?
  - [  ] Have I provided screenshots/videos of the changes?

### CODE REVIEW

File: src/locales/ko/dialogue.ts [hunk 1]
```diff
@@ -997,39 +997,39 @@ export const PGMdialogue: DialogueTranslationEntries = {
   },
   "chili": {
     "encounter": {
-      1: "Yeeeeooow! Time to play with FIRE!! I'm the strongest of us brothers!",
-      2: "Ta-da! The Fire-type scorcher Chili--that's me--will be your opponent!",
-      3: "I'm going to show you what me and my blazing Fire types can do!"
+      1: "이얏호-! 형제중 가장강한 나하고 놀자고!",
+      2: "짜잔! 불꽃타입 불꽃의 남자 팟, 바로 너의 상대가되어주지!",
```
Comment (returntoice) [inline src/locales/ko/dialogue.ts:1001]:
```suggestion
      1: "이얏호-! 형제 중 가장 강한 나하고 놀자고!",
      2: "짜잔! 불꽃타입 불꽃의 남자 팟, 바로 너의 상대가 되어주지!",
```

File: src/locales/ko/dialogue.ts [hunk 2]
```diff
@@ -997,39 +997,39 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
+      1: "졌지만… 모 두 불 태 웠 어!!",
+      2: "와우! 불태워졌네!",
+      3: "으악! 너가 나를 이겨버렸잖아!"
     },
     "defeat": {
-      1: "I'm on fire! Play with me, and you'll get burned!",
-      2: "When you play with fire, you get burned!",
-      3: "I mean, c'mon, your opponent was me! You didn't have a chance!"
+      1: "난 지금 불타오르고 있어, 나와 싸우게된다면 넌 다치게될거야!",
+      2: "불장난하면 밤에 이불에 지도를 그리는법이지!",
```
Comment (returntoice) [inline src/locales/ko/dialogue.ts:1011]:
```suggestion
      1: "난 지금 불타오르고 있어, 나와 싸우게 된다면 넌 다치게 될 거야!",
      2: "불장난하면 밤에 이불에 지도를 그리는 법이지!",
```

File: src/locales/ko/dialogue.ts [hunk 3]
```diff
@@ -997,39 +997,39 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
     }
   },
   "cilan": {
     "encounter": {
-      1: `Nothing personal... No hard feelings... Me and my Grass-type Pokémon will...
-               $Um... We're gonna battle come what may.`,
-      2: "So, um, if you're OK with me, I'll, um, put everything I've got into being, er, you know, your opponent.",
-      3: "OK… So, um, I'm Cilan, I like Grass-type Pokémon."
+      1: `개인적으로 받아들이지 마세요… 감정 상하지도 말고요… 저와 제 풀타입 포켓몬들 이라면…
+               $음… 무슨일이 있더라도 배틀을 할겁니다.`,
```
Comment (returntoice) [inline src/locales/ko/dialogue.ts:1018]:
```suggestion
      1: `개인적으로 받아들이지 마세요… 감정 상하지도 말고요… 저와 제 풀타입 포켓몬들이라면…
               $음… 무슨 일이 있더라도 배틀을 할 겁니다.`,
```

File: src/locales/ko/dialogue.ts [hunk 4]
```diff
@@ -997,39 +997,39 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
     },
     "victory": {
-      1: "Er… Is it over now?",
-      2: `…What a surprise. You are very strong, aren't you? 
-               $I guess my brothers wouldn't have been able to defeat you either…`,
-      3: "…Huh. Looks like my timing was, um, off?"
+      1: "…에- 그게…  끝나버렸나요?",
+      2: `…놀랐습니다. 당신 아주 강하군요. 
+               $팟이나 콘이라도 이길 수 없었을 것 같습니다…`,
+      3: "…어라, 타이밀이좀 잘못된것 같네요?"
```
Comment (returntoice) [inline src/locales/ko/dialogue.ts:1026]:
```suggestion
      3: "…어라, 타이밍이 좀 잘못된 것 같네요?"
```

File: src/locales/ko/dialogue.ts [hunk 5]
```diff
@@ -997,39 +997,39 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
     },
     "defeat": {
-      1: "Huh? Did I win?",
-      2: `I guess… 
-                $I suppose I won, because I've been competing with my brothers Chili and Cress, and we all were able to get tougher.`,
-      3: "It…it was quite a thrilling experience…"
+      1: "어라? 제가 이긴건가요?",
```
Comment (returntoice) [inline src/locales/ko/dialogue.ts:1029]:
```suggestion
      1: "어라? 제가 이긴 건가요?",
```

File: src/locales/ko/dialogue.ts [hunk 6]
```diff
@@ -997,39 +997,39 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
+      2: `제 생각엔… 
+                $아무래도 제가 이겼어야만 했을거라 생각합니다, 팟과 콘이랑 배틀하면서 우리는 모두 더 강해졌기 때문이죠.`,
```
Comment (returntoice) [inline src/locales/ko/dialogue.ts:1031]:
```suggestion
                $아무래도 제가 이겼어야만 했을 거라 생각합니다, 팟과 콘이랑 배틀하면서 우리는 모두 더 강해졌기 때문이죠.`,
```

File: src/locales/ko/dialogue.ts [hunk 7]
```diff
@@ -1203,13 +1203,13 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
-      1: "That is correct! It shall be I and my esteemed Water types that you must face in battle!"
+      1: "맞습니다! 당신은 저와 제 자랑스러운 물타입 포켓몬들과 상대하게 될것입니다."
     },
     "victory": {
-      1: "Lose? Me? I don't believe this."
+      1: "곤란하네… 이 콘이 질 줄이야."
     },
     "defeat": {
-      1: "This is the appropriate result when I'm your opponent."
+      1: "제가 당신의 상대였기때문에 이건 당연한 결과였습니다."
```
Comment (returntoice) [inline src/locales/ko/dialogue.ts:1212]:
```suggestion
      1: "제가 당신의 상대였기 때문에 이건 당연한 결과였습니다."
```

**Review by returntoice — CHANGES_REQUESTED** [review]
아마 원본의 오류 같기는 하지만, 맞춤법이 틀린 건 고치는 게 좋을 것 같아 수정했습니다.

**Comment by soy0ka** [comment]
일부는 블랙 화이트 게임내의 대사를 그대로 가져오면서 발생한 문제 같습니다 수정 감사드립니다

**Review by returntoice — APPROVED** [review]

**Review by Tempo-anon — DISMISSED** [review]

File: src/locales/ko/dialogue.ts [hunk 8]
```diff
@@ -997,39 +997,39 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
   },
   "cilan": {
     "encounter": {
-      1: `Nothing personal... No hard feelings... Me and my Grass-type Pokémon will...
-               $Um... We're gonna battle come what may.`,
-      2: "So, um, if you're OK with me, I'll, um, put everything I've got into being, er, you know, your opponent.",
-      3: "OK… So, um, I'm Cilan, I like Grass-type Pokémon."
+      1: `개인적으로 받아들이지 마세요… 감정 상하지도 말고요… 저와 제 풀타입 포켓몬들 이라면…
+               $음… 무슨일이 있더라도 배틀을 할겁니다.`,
+      2: "네. 저로 괜찮다면 진심을 담아서 상대해 드리겠습니다.",
```
Comment (Tempo-anon) [inline src/locales/ko/dialogue.ts:1019]: There's a trailing space here that's causing the linter to fail

**Review by Tempo-anon — APPROVED** [review]

**Pull request merged**